leapp/cli/upgrade/__init__.py

```diff
@@ -211,11 +210,12 @@ def preupgrade(args):
 
     report_txt, report_json = [os.path.join(get_config().get('report', 'dir'),
                                             'leapp-report.{}'.format(f)) for f in ['txt', 'json']]
-    report_info([report_txt, report_json], fail=workflow.errors)
     # fetch all report messages as a list of dicts
-    messages = fetch_upgrade_report_raw(context, renderers=False)
-    with open(report_json, 'w+') as f:
-        json.dump({'entries': messages}, f, indent=2)
+    messages = fetch_upgrade_report_messages(context)
+    generate_report_file(messages, report_json)
+    # XXX FIXME original actor filtered all non-high messages leaving high/indicator only. Should we do the same?
```

Comment:
Definitely not. This has been a mistake which we need to fix...

Reply:
Thanks, will remove the XXX FIXME then. Now 2 files (txt and json) finally match